docs/opensauced-packages/semantic-release.md

```diff
@@ -131,39 +131,39 @@ jobs:
           echo ${{ steps.semantic-release.outputs.release-version }}
 ```
 
-## 📦 NPM usage
+## 📦 `npm` Usage
 
 You can opt to use this package in your local tooling. Proceed as you would normally would, replacing `npm` with your package manager of choice and install the package:
 
-```bash
+```shell
 npm install --save-dev @open-sauced/semantic-release-conventional-config
 ```
 
-The shareable config can then be configured in the [**semantic-release** configuration file](https://github.com/semantic-release/semantic-release/blob/master/docs/usage/configuration.md#configuration):
+The shareable config can then be configured in the [  `semantic-release` configuration file](https://github.com/semantic-release/semantic-release/blob/master/docs/usage/configuration.md#configuration):
 
 ```json
 {
   "extends": "@open-sauced/semantic-release-conventional-config"
 }
 ```
 
-Now all you need to do is create a release:
+Now, all you need to do is create a release:
 
 ```shell
 npx semantic-release
 ```
 
 ## 🔧 Configuration
 
-See each [plugin](#-plugins) documentation for required installation and configuration steps.
+See each [plugin](#-plugins) documentation for the required installation and configuration steps.
```

Comment:
The hyperlink here is going no where. And we don't have a section called "Plugins".
Should we remove this?

Reply:
for sure.